src/api/procedures/createCheckpointSchedule.ts

```diff
@@ -18,8 +18,17 @@ import {
 import { filterEventRecords } from '~/utils/internal';
 
 export interface CreateCheckpointScheduleParams {
+  /**
+   * The date for when to begin creating snapshots. A null value indicates immediately
+   */
   start: Date | null;
+  /**
+   * The cadence in which to make Checkpoints. A null value indicates to create only one Checkpoint, regardless of repetitions specified
+   */
   period: CalendarPeriod | null;
```

Comment:
To me a null period doesn't make sense when there is an API to create a single Checkpoint.

Reply:
This allows you to schedule its creation in the future

Reply:
But yeah, null date + null period is the same as just creating a Checkpoint directly

Reply:
Oh, I didn't think about that. I guess it does give new functionality then.

Reply:
I moved it to a @note, + specified that use case.